applications/fax/src/fax_smtp.erl

```diff
@@ -458,6 +459,82 @@ maybe_faxbox_owner(#state{faxbox=FaxBoxDoc}=State) ->
             end
     end.
 
+-spec maybe_faxbox_domain(state()) -> state().
+maybe_faxbox_domain(#state{faxbox_email=Domain}=State) ->
+    ViewOptions = [{'key', Domain}],
+    case couch_mgr:get_results(?WH_ACCOUNTS_DB, <<"accounts/listing_by_realm">>, ViewOptions) of
+        {'ok', [JObj]} -> AccountId = wh_json:get_value([<<"value">>, <<"account_id">>], JObj),
+                          maybe_faxbox_by_owner_email(AccountId, State);
+        _ -> 
```

Comment:
Is there any value in differentiating `{'ok', []}` vs `{'error', _E}` in the case clauses?